<?xml version="1.0"?>
<launch>
    <!-- Load the parameters for the SARSA algorithm -->
    <rosparam command="load" file="$(find my_gurdy_description)/config/params_sarsa.yaml" />
    
    <!-- Launch the training node -->
    <node pkg="my_gurdy_description" type="train_gurdy_sarsa.py" name="gurdy_sarsa" output="screen"/>
</launch>